### PR TITLE
RequesterParams implementation

### PR DESCRIPTION
**Pull request by kuznetsovmoci**

### CODE REVIEW

File: rosidl_typesupport_opendds_cpp/include/rosidl_typesupport_opendds_cpp/requester_parameters.h [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+
+#include <string>
+#include "dds/DCPS/Service_Participant.h"
+#include "dds/DCPS/Marked_Default_Qos.h"
+
+  class RequesterParams {
+  public:
+    RequesterParams()
+      : participant_(nullptr) {
+    };
```
Comment (mitza-oci) [inline rosidl_typesupport_opendds_cpp/include/rosidl_typesupport_opendds_cpp/requester_parameters.h]:
```suggestion
    }
```

File: rosidl_typesupport_opendds_cpp/include/rosidl_typesupport_opendds_cpp/requester_parameters.h [hunk 2]
```diff
@@ -0,0 +1,110 @@
+#ifndef ROSIDL_TYPESUPPORT_OPENDDS_CPP__REQUESTER_PARAMETERS_H_
+#define ROSIDL_TYPESUPPORT_OPENDDS_CPP__REQUESTER_PARAMETERS_H_
+
+#include <string>
+#include "dds/DCPS/Service_Participant.h"
+#include "dds/DCPS/Marked_Default_Qos.h"
+
+  class RequesterParams {
```
Comment (mitza-oci) [inline rosidl_typesupport_opendds_cpp/include/rosidl_typesupport_opendds_cpp/requester_parameters.h]: Should this belong to a namespace?

File: rosidl_typesupport_opendds_cpp/include/rosidl_typesupport_opendds_cpp/requester_parameters.h [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+  public:
+    RequesterParams()
+      : participant_(nullptr) {
+    };
+
+    RequesterParams(DDS::DomainParticipant* participant) {
```
Comment (mitza-oci) [inline rosidl_typesupport_opendds_cpp/include/rosidl_typesupport_opendds_cpp/requester_parameters.h]:
```suggestion
    explicit RequesterParams(DDS::DomainParticipant* participant) {
```

**Review by adamsj-ros — APPROVED** [review]

**Review by mitza-oci — CHANGES_REQUESTED** [review]

File: rosidl_typesupport_opendds_cpp/include/rosidl_typesupport_opendds_cpp/replier_parameters.h [hunk 4]
```diff
@@ -0,0 +1,115 @@
...
+#include <string>
+#include "dds/DCPS/Service_Participant.h"
+#include "dds/DCPS/Marked_Default_Qos.h"
+
+namespace rosidl_typesupport_opendds_cpp
+{
+// TODO: may need to be templated, if it needs a listener: http://community.rti.com/rti-doc/510/ndds/doc/html/api_cpp/classconnext_1_1ReplierParams.html
+class ReplierParams {
+public:
+  explicit ReplierParams()
```
Comment (mitza-oci) [inline rosidl_typesupport_opendds_cpp/include/rosidl_typesupport_opendds_cpp/replier_parameters.h]:
```suggestion
  ReplierParams()
```

File: rosidl_typesupport_opendds_cpp/include/rosidl_typesupport_opendds_cpp/replier_parameters.h [hunk 5]
```diff
@@ -0,0 +1,115 @@
...
+    : participant_(nullptr) {
+  }
+
+  ReplierParams(DDS::DomainParticipant* participant) {
```
Comment (mitza-oci) [inline rosidl_typesupport_opendds_cpp/include/rosidl_typesupport_opendds_cpp/replier_parameters.h]:
```suggestion
  explicit ReplierParams(DDS::DomainParticipant* participant) {
```

Comment (mitza-oci) [inline rosidl_typesupport_opendds_cpp/include/rosidl_typesupport_opendds_cpp/replier_parameters.h]: If we're going to use `_var`s for other objects, let's use it here too

**Review by mitza-oci — CHANGES_REQUESTED** [review]

**Pull request merged**